Dockerfile

```diff
@@ -3,9 +3,10 @@ FROM alpine:3.10
 ARG ARCH=
 
 ENV K3S_RELEASE https://github.com/rancher/k3s/releases/download/${DRONE_TAG}/k3s${ARCH}
+ENV K3S_RELEASE_CHECKSUM https://github.com/rancher/k3s/releases/download/${DRONE_TAG}/sha256sum${ARCH}.txt
 
-RUN wget -O /bin/k3s ${K3S_RELEASE}
-RUN chmod +x /bin/k3s
+RUN wget -O /opt/k3s ${K3S_RELEASE}
+RUN chmod +x /opt/k3s
```

Comment:
Why the change to `/opt`? Assuming there is a good reason for the prefix change, shouldn't this be `/opt/bin/k3s`?

Reply:
no particular reason I just wanted the binary to be placed in /opt to be checked first against the hash and then copied to the right location